component/firecracker-base/orchestrate-install.sh

```diff
@@ -133,7 +139,7 @@ execute_configuration_management() {
         cp ./jailer /usr/bin/jailer
 
         # Load kernel module
-        modprobe kvm_intel
+        modprobe kvm_intel || echo "loading AMD instead" && modprobe kvm_amd
```

Comment:
nit: I'd probably prefer we remove the log line as we're logging to stdout/stderr anyway & it'll bomb out

Reply:
modprobe only logs if it fails, so you get a log line saying it failed and then nothing else after. The result of the change here is if it fails to install intel, it will tell you and then move on. If it succeeds, it'll just move on.